libsql-categorizer/src/lib.rs

```diff
@@ -0,0 +1,245 @@
+extern crate pest;
+#[macro_use]
+extern crate pest_derive;
```

Comment:
This is rust 2023 we don't need these anymore 😄

Reply:
I did wonder for a short while why it's there... but since I started the development with a copy-paste of one of the pest examples :shrug: will drop, thx